xbmc/pvr/windows/GUIWindowPVRBase.cpp

```diff
@@ -151,6 +162,9 @@ void CGUIWindowPVRBase::OnInitWindow(void)
     CGUIWindow::OnInitWindow(); // do not call CGUIMediaWindow as it will do a Refresh which in no case works in this state (no channelgroup!)
     ShowProgressDialog(g_localizeStrings.Get(19235), 0); // PVR manager is starting up
   }
+
+  // This has to be done after base class OnInitWindow to restore correct selection
+  InitChannelGroupsControl();
```

Comment:
This should be moved to the first condition `if (InitChannelGroup())` block in this method.

Reply:
Yep, thanks. Does not harm where it is now, but fits better in the first block.